courses/free-courses-bn.md

```diff
@@ -240,6 +240,7 @@
 #### React
 
 * [১ ভিডিওতে রিয়্যাক্ট শিখুন ! ফুল কোর্স](https://www.youtube.com/watch?v=6wilewRV3xQ) - Rabbil Hasan
+* [বাংলা REACT JS  টিউটোরিয়াল \| REACT full course \| REACT TUTORIAL in BANGLA \| REACT js in Bangla](https://youtube.com/playlist?list=PLvT67qI_dN-LPi-_6sTrxdqdKvMGeEh8N) - IT VAYA
```

Comment:
```suggestion
* [বাংলা REACT JS  টিউটোরিয়াল \| REACT full course \| REACT TUTORIAL in BANGLA \| REACT js in Bangla](https://www.youtube.com/playlist?list=PLvT67qI_dN-LPi-_6sTrxdqdKvMGeEh8N) - IT VAYA
```
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes